include/libnuraft/peer.hxx

```diff
@@ -317,6 +318,22 @@ public:
         last_streamed_log_idx_.store(0);
     }
 
+    size_t get_flying_bytes() {
+        return flying_bytes.load();
+    }
+
+    void flying_bytes_add(size_t total_size) {
+        flying_bytes.fetch_add(total_size);
+    }
+
+    void flying_bytes_sub(size_t total_size) {
+        flying_bytes.fetch_sub(total_size);
```

Comment:
Can you add the below
```c++
assert(flying_bytes >= 0);
```
right after this line? so as to catch any bug that corrupts this number during the test.